.circleci/codebuild-checkout.sh

```diff
@@ -21,6 +21,10 @@ if [[ "$CODEBUILD_WEBHOOK_TRIGGER" =~ ^pr/ || "$CODEBUILD_SOURCE_VERSION" =~ ^pr
   echo "Creating temporary local branch for PR build"
   TEMP_BRANCH_NAME=$(cat /proc/sys/kernel/random/uuid)
   git checkout -b $TEMP_BRANCH_NAME
+elif [[  "$CODEBUILD_WEBHOOK_TRIGGER" == "branch/dev" ]]; then
+  # We're in E2E workflow triggered after pushing to dev.
+  echo "Checking out dev"
```

Comment:
nit: there's a `set -v` magic to show echo the command https://stackoverflow.com/questions/2853803/how-to-echo-shell-commands-as-they-are-executed

Reply:
I find it ugly...
![image](https://github.com/aws-amplify/amplify-cli/assets/5849952/643b94b6-f696-4b32-8f0c-be6b43ff14d4)

Reply:
😂